spec/jstests-spec.js

```diff
@@ -672,4 +673,189 @@ describe("Checking binary search functions", function () {
     expect(prevLower).toBe(-1);
     expect(nextHigher).toBe(0);
   });
-}); 
\ No newline at end of file
+});
+
+describe("Testing slider functionality", function () {
+  beforeAll(function () {
+    document.body.innerHTML = '';
+  });
+
+  beforeEach(function () {
+    document.body.innerHTML = `
+    <div id="slider" class="mdc-slider mdc-slider--discrete mdc-slider--display-markers" tabindex="0" role="slider" aria-valuemin="0" aria-valuemax="0" aria-valuenow="0" aria-label="Select Value">
```

Comment:
do you think it could be helpful to import this string as a constant?

Reply:
I just copy pasted this from the material design slider reference. Do you think it would be weird to put it in script given that it doesn't have much to do with our code?

Reply:
Haha you're probably right it's fine!